doc/man/8/mount.ceph.rst

```diff
@@ -199,6 +199,39 @@ Advanced
     option is enabled, a namespace operation may complete before the MDS
     replies, if it has sufficient capabilities to do so.
 
+:command:`crush_location=x`
+    Specify the location of the client in terms of CRUSH hierarchy.  This
+    is a set of key-value pairs separated from each other by '|', with keys
+    separated from values by ':'.  Note that '|' may need to be quoted or
+    escaped to avoid it being interpreted as a pipe by the shell. The key
+    is the bucket type name (e.g. rack, datacenter or region with default
+    bucket types) and the value is the bucket name. For example, to indicate
+    that the client is local to rack "myrack", data center "mydc" and region
+    "myregion"::
+
+      crush_location=rack:myrack|datacenter:mydc|region:myregion
+
+    Each key-value pair stands on its own: "myrack" doesn't need to reside in
+    "mydc", which in turn doesn't need to reside in "myregion".  The location
+    is not a path to the root of the hierarchy but rather a set of nodes that
+    are matched independently.
```

Comment:
Since this is a cut-and-paste from RBD documentation, I'm wondering why the note about multipath locations was dropped?
> "Multipath" locations are supported, so it is possible to indicate locality for multiple parallel hierarchies:
>
>    crush_location=rack:myrack1|rack:myrack2|datacenter:mydc

Reply:
I couldn't remember why this is dropped, and added it back.